<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetTreeLocationDialog</class>
 <widget class="QDialog" name="SetTreeLocationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>517</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>445</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CFD study/case localization or study creation</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="checkBoxLoad">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Load existing study or case</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxCreate">
        <property name="text">
         <string>Create study</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="StudyGB" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="StudyDirLabel">
        <property name="minimumSize">
         <size>
          <width>95</width>
          <height>33</height>
         </size>
        </property>
        <property name="text">
         <string>Study/Case location</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>StudyDirName</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="StudyDirName">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="NameLabel">
        <property name="minimumSize">
         <size>
          <width>95</width>
          <height>33</height>
         </size>
        </property>
        <property name="text">
         <string>Study/Case name</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>StudyLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="StudyLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>New study directory name created</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="CaseWidget" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Cases</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="CaseLineEdit">
           <property name="toolTip">
            <string>Input cases names separated with space</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QRadioButton" name="radioButtonNeptune">
           <property name="text">
            <string>NEPTUNE_CFD</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonSaturne">
           <property name="text">
            <string>Code_Saturne</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxCopyFrom">
        <property name="text">
         <string>Copy from existing case</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="copyFromCase_widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="copyFromCase">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="SyrthesWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="checkBoxCouplingSaturneSyrthes">
           <property name="text">
            <string>Coupling with Syrthes</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="SyrthesGroupBox" native="true">
           <layout class="QGridLayout" name="gridLayout_2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="0" column="1">
             <widget class="QLineEdit" name="syrthesCase">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>33</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Input cases names separated with space</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="nprocsLabel">
              <property name="minimumSize">
               <size>
                <width>89</width>
                <height>33</height>
               </size>
              </property>
              <property name="text">
               <string>Nprocs</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
              <property name="buddy">
               <cstring>NprocsLineEdit</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="NprocsLineEdit">
              <property name="minimumSize">
               <size>
                <width>146</width>
                <height>33</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Input cases names separated with space</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="SyrthesCaseLabel">
              <property name="minimumSize">
               <size>
                <width>89</width>
                <height>33</height>
               </size>
              </property>
              <property name="text">
               <string>Syrthes case</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
              <property name="buddy">
               <cstring>syrthesCase</cstring>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>checkBoxLoad</tabstop>
  <tabstop>checkBoxCreate</tabstop>
  <tabstop>StudyDirName</tabstop>
  <tabstop>StudyLineEdit</tabstop>
  <tabstop>CaseLineEdit</tabstop>
  <tabstop>radioButtonNeptune</tabstop>
  <tabstop>radioButtonSaturne</tabstop>
  <tabstop>checkBoxCopyFrom</tabstop>
  <tabstop>copyFromCase</tabstop>
  <tabstop>checkBoxCouplingSaturneSyrthes</tabstop>
  <tabstop>syrthesCase</tabstop>
  <tabstop>NprocsLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SetTreeLocationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>503</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SetTreeLocationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>494</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxCouplingSaturneSyrthes</sender>
   <signal>clicked(bool)</signal>
   <receiver>SyrthesGroupBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>400</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonNeptune</sender>
   <signal>clicked(bool)</signal>
   <receiver>SyrthesWidget</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>415</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonSaturne</sender>
   <signal>clicked(bool)</signal>
   <receiver>SyrthesWidget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>415</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxCreate</sender>
   <signal>clicked(bool)</signal>
   <receiver>copyFromCase_widget</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxLoad</sender>
   <signal>clicked(bool)</signal>
   <receiver>CaseWidget</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>310</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxCreate</sender>
   <signal>clicked(bool)</signal>
   <receiver>CaseWidget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>310</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxCreate</sender>
   <signal>clicked(bool)</signal>
   <receiver>copyFromCase_widget</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxLoad</sender>
   <signal>clicked(bool)</signal>
   <receiver>copyFromCase_widget</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
